src/main/kotlin/exercise2/task5/CreateUserDSL.kt

```diff
@@ -49,11 +49,16 @@ internal data class Address(
  */
 
 internal fun user(initUser: User.() -> Unit): User {
-    TODO("Implement me!!!")
+    val user = User()
```

Comment:
That is it! We can also write this with the scoped functions:
```suggestion
    return User().apply(initUser)
```